corehq/apps/domain/migrations/0015_domainoptiontoggles.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations, models
+
+from corehq.apps.domain.models import DomainOptionToggles
+from corehq.toggles import NAMESPACE_DOMAIN, Toggle
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+def get_enabled_domains():
+    toggle = Toggle.cached_get('use_latest_build_cloudcare')
+    if not toggle:
+        return []
+    prefix = NAMESPACE_DOMAIN + ':'
+    skip = len(prefix)
+    return [d[skip:] for d in toggle.enabled_users if d.startswith(prefix)]
+
+
+@skip_on_fresh_install
+def _upsert_domain_option_toggles(apps, schema_editor):
+    for domain in get_enabled_domains():
+        DomainOptionToggles.objects.update_or_create(
```

Comment:
I didn't really need to use `update_or_create()`, but this pattern makes it easier to reuse if we want to add more fields to the model.